src/App.tsx

```diff
@@ -1,14 +1,52 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [filtered, setFiltered] = useState(Filters.ALL);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [filter, setFilter] = useState(Filters.ALL);
```
FYI: for boolean variables: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig